core-services/prow/02_config/_config.yaml

```diff
@@ -1085,6 +1081,42 @@ tide:
     - do-not-merge/invalid-owners-file
     repos:
     - openshift-kni/baremetal-deploy
+  - includedBranches:
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    labels:
+    - lgtm
+    - approved
+    - bugzilla/valid-bug
+    - cherry-pick-approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/blocked-paths
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
+    repos:
+    - openshift-kni/performance-addon-operators
+    - openshift-kni/cnf-features-deploy
+  - excludedBranches:
+    - release-4.3
+    - release-4.4
+    - release-4.5
+    labels:
+    - lgtm
+    - approved
+    missingLabels:
+    - needs-rebase
+    - do-not-merge/blocked-paths
+    - do-not-merge/hold
+    - do-not-merge/work-in-progress
+    - do-not-merge/invalid-owners-file
+    - bugzilla/invalid-bug
+    repos:
+    - openshift-kni/performance-addon-operators
+    - openshift-kni/cnf-features-deploy
```

Comment:
i think the baremetal-deploy repo probably needs to be under here as well, or they won't be able to merge.

Reply:
they have their own single config already, without excluded/included branches, and without the valid-bug label requirement, see just above this.